scripts/check-npm.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-if [[ `npm -v` != 3* ]]
-  then
-    npm install --global npm
-fi
```

Comment:
`check-node-version` does not install the latest npm version right? This script is in place so that Travis works with Node 4, though I don't remember exactly why.